### DISCUSSION TITLE
How is the max height calculated when using a heightmap?

### DISCUSSION THREAD

**Discussion by antzGames** [JamesTKhan/Mundus#169] [Q&A] [answered]
So my example heightmap and terrain generated shown below.  

Black RGB (0,0,0) becomes a height of 0.
White RGB (255,255,255) becomes a height of 272???

**Why is the maximum height 272**? Is alpha involved?

Should we introduce a scaling option (min/max) values?

![hieghmap](https://user-images.githubusercontent.com/10563814/237049291-37b03a25-a32a-413a-a0e7-676660ca8341.png)

![image](https://user-images.githubusercontent.com/10563814/237051517-e6c09424-954f-40ae-9ee0-2fb475acd8bd.png)

**Comment by JamesTKhan** [comment, marked as answer]
With heightmaps it looks like the max height is calculated using an arbitrary value, terrainWidth * 0.17f

It should be possible to plug in different values to replace the hard coded 0.17f, the UI fields just have to be added in. 

Adding in fields to change the value makes sense to me.


https://github.com/JamesTKhan/Mundus/blob/master/editor/src/main/com/mbrlabs/mundus/editor/ui/modules/inspector/components/terrain/generation/HeightmapTab.kt#L75
